### ISSUE TITLE
Fail when starting from an empty string in one of the multi-rules

### ISSUE THREAD

**Issue by maxitg** [maxitg/SetReplace#129]
The following fails with messages:
```
In[] := SetReplace[{}, ToPatternRules[{{1} -> {}, {} -> {}}]]
```
![image](https://user-images.githubusercontent.com/1479325/69012714-ddb1d000-0946-11ea-9137-b86b7e6ce1f5.png)

**Version (please complete the following information):**
 - OS: macOS 10.15.1 (19B88)
 - Wolfram Language Version: "12.1.0 for Mac OS X x86 (64-bit) (November 6, 2019)"
 - SetReplace git sha: cb1a2de482afaa1b05ebe2c48bb00d318751f562

**Comment by daneelsan** [comment]
In:
```wl
In[] := SetReplace[{}, ToPatternRules[{{1} -> {}, {} -> {}}]]
```
Is it intended that two events occur? `{{{1, {} -> {}, 1}, {2, {} -> {}, 1}}}`

When the input is just:
```wl
In[] := SetReplace[{}, ToPatternRules[{{} -> {}}]]
```
the only event that occurs is `{{{1, {} -> {}, 1}}}`

![image](https://user-images.githubusercontent.com/40190339/95923601-77624700-0d7b-11eb-9ef7-6302b80cf81a.png)

**Comment by maxitg** [comment]
No, `SetReplace` should do one event at a time, i.e., it's equivalent to `<|"MaxEvents" -> 1|>` (without the third argument).
